packages/insomnia-app/app/ui/components/wrapper-onboarding.tsx

```diff
@@ -119,15 +124,25 @@ class WrapperOnboarding extends PureComponent<Props, State> {
     const {
       settings: { enableAnalytics },
     } = this.props.wrapperProps;
+
+    const isControlledByInsomniaConfig = (
+      getControlledStatus(this.props.wrapperProps.settings)('enableAnalytics')?.controller === 'incognitoMode'
+    );
+
+    const analyticsMessage = enableAnalytics
+      ? `Thanks for helping make ${getAppName()} better!`
+      : `Opted out of analytics${isControlledByInsomniaConfig ? ' by Incognito Mode' : ''}`;
```

Comment:
```suggestion
      : `Opted out of analytics${isControlledByInsomniaConfig ? ' by incognito mode' : ''}`;
```

@wdawson go ahead and commit this suggestion if you'd like the casing to be different.  I wasn't sure which is best.

Reply:
this no longer applies, since 0452c96500b53c445cf2f3b54e3c0b4a9ec93b16